store.go

```diff
@@ -244,6 +247,7 @@ func (s *store) handleSyncError(err error, isColdStart bool) {
 }
 
 func (s *store) fetchConfigSpecsFromServer(isColdStart bool) {
+	s.logProcess("Loading specs from network...")
```

Comment:
just realized this is going to run for every d_c_s call - I think we should only output this for the initial one

Reply:
I think it could be useful to get diagnostics for each sync. Maybe make this an option then?